src/geoengine/collections.md

```diff
@@ -0,0 +1,17 @@
+# Layer Collections
+
+Collections are groups of [`Layers`](./layers.md).
+Collections themselves can be grouped inside other collections.
```

Comment:
We should explain that Collections and layers can be contained in multiple collections.

I would also just make one chapter that explain layer and collections in one page. Or with two subchapters,

Reply:
[96f675f](https://github.com/geo-engine/docs/pull/44/commits/96f675f2c13a7be11b8e5ca8825ea0c60c79f1b9)